<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderUI</class>
 <widget class="QWidget" name="OrderUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QTableView" name="MainTableView"/>
     <widget class="QToolBox" name="OToolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="OInput">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>281</width>
         <height>554</height>
        </rect>
       </property>
       <attribute name="label">
        <string>order add/delete</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="IDlabel">
            <property name="text">
             <string>order ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="IDLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="CustomerLabel">
            <property name="text">
             <string>customer ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="CustomerLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="ProductLabel">
            <property name="text">
             <string>product ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ProductLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="ProductCntLabel">
            <property name="text">
             <string>product count</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ProductCntLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>194</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="AddButton">
          <property name="text">
           <string>add/change order</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DeleteButton">
          <property name="text">
           <string>delete order</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>194</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="OSearch">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>281</width>
         <height>554</height>
        </rect>
       </property>
       <attribute name="label">
        <string>order search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="SearchTableView"/>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <widget class="QComboBox" name="SearchComboBox">
           <item>
            <property name="text">
             <string>oid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>pid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>pcnt</string>
            </property>
           </item>
          </widget>
          <widget class="QLineEdit" name="SearchLineEdit"/>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SearchButton">
          <property name="text">
           <string>search</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
